### ISSUE TITLE
Opting store pages of Google's "Web Light"

### ISSUE THREAD

**Issue by sybrew** [woocommerce/woocommerce#22525]
See Google's [Web Light documentation](https://support.google.com/webmasters/answer/6211428?hl=en).

Web Light is a solution brought by Google to slim down your pages (automagically) when the visitor (from Google Search) is visiting from a slow mobile device.

To test this, use this URL in your browser: `http://googleweblight.com/i?u=[your_website_URL]`

**Is your feature request related to a problem? Please describe.**
Yes, WooCommerce and Web Light don't work well together. WooCommerce requires dynamic rendering of the cart and store items.

**Describe the solution you'd like**
Google provides an opt-out header to use. From their documentation, it's this: `Cache-Control: no-transform`

**Additional context**
WordPress and caching plugins might also send out a header. We'd need to figure out how to combine these headers, without conflict. Note that this header generally isn't necessary on non-WC pages; so, consider not opting out the whole site.

**Comment by sybrew** [comment]
Thanks for looking into this! I've tested the following problems using the Twenty Seventeen theme.

For example, when using the "Add to cart" button from an archive, the page tells that the product has been added to the cart.
![image](https://user-images.githubusercontent.com/7662236/51757000-9d851680-20c2-11e9-9671-ba43c8b8bd7d.png)

...but the cart page tells it's empty after following the "View cart" link.

Another example, when pressing an (enumerated) "Add to cart" button from a product page, the visitor gets redirected from Google Web Light to the product page (without adding anything to the cart).
I'm not sure if a real (non-simulated) Web Light experience would entail this behavior.

![image](https://user-images.githubusercontent.com/7662236/51757095-d2916900-20c2-11e9-8572-04e4497ca25d.png)

Another scenario where I expect faults to occur would be on checkout pages with custom payment gateways; alas, I haven't gotten this far yet.

I am curious to see if Web Light will fix these issues, so we don't have to. Web Light's usage isn't widespread (I get eight queries to my site every three months...), so there might be things for Google to iron out first.
